spec/abstractops.html

```diff
@@ -108,17 +108,14 @@ <h1>DurationHandleFractions ( _fHours_, _minutes_, _fMinutes_, _seconds_, _milli
         1. Set _seconds_ to floor(_secs_).
         1. Let _fSeconds_ be _secs_ modulo 1.
         1. If _fSeconds_ is not equal to 0, then
-          1. If any of _milliseconds_, _microseconds_, _nanoseconds_ is not 0, throw a *RangeError* exception.
```

Comment:
is this not normative?

Reply:
I don't believe so — it's dead code, these checks are already performed in step 2a, and the variables are not changed in the meantime.